Thirdlesson/ObjectTracking /ObjectCatch.cpp

```diff
@@ -0,0 +1,18 @@
+//
+// Created by s1000633 on 5/18/21.
+//
+
+#include "ObjectCatch.h"
+
+ObjectCatch::ObjectCatch(const cv::Mat &image, const cv::Scalar& lower_b, const cv::Scalar& upper_b) {
```

Comment:
what if we want to reuse this code for catching other objects? should we create a new objects? Or maybe its better to move this to separate function that can be reused?